yb-voyager/src/utils/utils.go

```diff
@@ -431,7 +431,7 @@ func GetRedactedURLs(urlList []string) []string {
 	for _, u := range urlList {
 		obj, err := url.Parse(u)
 		if err != nil {
-			ErrExit("invalid URL: %q", u)
```

Comment:
I think yes, an error can include the URL in some cases, as I see `url.Parse()` is including the `rawURL` in the error msg. 
```
func Parse(rawURL string) (*URL, error) {
	...
	if err = url.setFragment(frag); err != nil {
		return nil, &Error{"parse", rawURL, err}
	}
```

Reply:
ok, what we can do here is instead of using ErrExit() -> log + console
We print different messages for log vs console. 
console can have error message(containing url)
log should have a generic msg like you added